web/assets/css/transmission-app.scss

```diff
@@ -390,6 +386,11 @@ $toolbar-height: $toolbar-height-number * 1px;
   }
 }
 
+#torrent-search {
```

Comment:
Just sheer curiosity, wondering why this was moved out of mainwin-filterbar — CSS and all.

Reply:
It was also under @include for-phone-only class which became obsoleted after a fix in another class outside it. I see that you like to have class under another class whenever possible, is that correct? I can move it to there if you'd like.

Reply:
@dareiff I want to close this as resolved, you're just asking a question or would you like me to put the id selector under there?

Reply:
I’d personally prefer it to be enclosed in the parent class (because sass and all that) but totally fine if you don’t want to make that change. Close ‘er.

Reply:
Finished moving it back into parent class!